### PR TITLE
Fixes a problem with the sequence number

### PR DESCRIPTION
**Pull request by ugocupcic**
Merge #131 first.

### CODE REVIEW

File: sr_ronex_utilities/scripts/plot_tcat_state.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+
+    print
+    base=[0,0,0,0]
+    for r in data.received_data:
+        base[i] = data.received_data[i].timestamp_ns - (data.received_data[i].FPI - data.received_data[i].first_sample_number)
+        #print base[i], data.received_data[i].timestamp_ns, data.received_data[i].first_sample_number, data.received_data[i].FPI
+        #ts = (data.received_data[i].timestamp_ns*(1000.0/15.65))
+        #print data.received_data[i].timestamp_ns, str(hex( int(ts) ))
+        i = i + 1
```
Comment (shadowmanos) [inline sr_ronex_utilities/scripts/plot_tcat_state.py]: lines 95 - 102 could simply be

base=[r.timestamp_ns - (r.FPI - r.first_sample_number) for r in data.received_data]

simpler, more efficient and safer if data.received_data is longer than 4

File: sr_ronex_utilities/scripts/plot_tcat_state.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+    base_min = min(base)
+    print base, base_min
+    
+    i = 0
+    for r in data.received_data:
+        base[i] = base[i] - base_min
+        i = i + 1
+    print base
```
Comment (shadowmanos) [inline sr_ronex_utilities/scripts/plot_tcat_state.py:99]: lines 107 - 111

print [b - base_min for b in base]

see previous comment

File: sr_ronex_utilities/scripts/plot_tcat_state.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        y1 = []
+        y2 = []
+        y3 = []
+        x  = []
+        x_value = base[receiver_index]
+
+        for impulse in receiver.impulse_response:
+            y1.append(impulse.real)
+            y2.append(impulse.imaginary)
+            y3.append(sqrt(impulse.real*impulse.real + impulse.imaginary*impulse.imaginary))
```
Comment (shadowmanos) [inline sr_ronex_utilities/scripts/plot_tcat_state.py]: y3.append(sqrt(impulse.real**2 + impulse.imaginary**2))

File: sr_ronex_utilities/scripts/plot_tcat_state.py (hunk 2 above)
Comment (ugocupcic) [inline sr_ronex_utilities/scripts/plot_tcat_state.py:99]: this doesn't update base[] does it? I need to do `base = [b - base_min for b in base]` don't I?

Comment (shadowmanos) [inline sr_ronex_utilities/scripts/plot_tcat_state.py:99]: Yes since apparently base is used later on

**Pull request merged**